spec/controllers/import_rdf_controller_spec.rb

```diff
@@ -0,0 +1,115 @@
+require 'rails_helper'
+
+RSpec.describe ImportRdfController, :type => :controller do
+  let(:logged_in) { true }
+  let(:form_factory) { class_double("ImportForm") }
+  let(:form) { instance_double("ImportForm") }
+
+  before do
+    allow(controller).to receive(:check_auth).and_return(true) if logged_in
+    allow(controller).to receive(:form_factory).and_return(form_factory)
+    allow(form_factory).to receive(:new).and_return(form)
+
+    expect(controller).not_to receive(:injector)
```

Comment:
Pretty much - trying to prove (more to myself, perhaps) that the object is fully isolated.  ... though maybe that warrants an injector test.  Thoughts?